### ISSUE TITLE
Feature request: warning when making a commit with 0 changes

### ISSUE THREAD

**Issue by Reonu** [godotengine/godot-git-plugin#220]
I keep forgetting to actually stage the changes and making commits with no changes at all. Maybe show the user a warning so that they can confirm if they're doing it on purpose?

**Comment by NostalgiaNinja** [comment]
Can an option be provided to stage all changes or cancel the commit so that we can stage the change when it happens? This would help with empty commits. (I had to make a duplicate commit as a result)

**Comment by perroboc** [comment]
A suggestion: Just a checkbox that allows empty commits. If empty commits are disabled, block the commit when no changes are staged.

**Comment by dorianmk** [comment]
Yeah, I'm also struggling with this one - message warning or option for blocking empty commits would be just fine

**Comment by Nodd** [comment]
I keep getting bitten by this. I see there was a use case indicated in #240, but it seems minimal compared to the nuisance of adding an empty commit by mistake. Please disable this or provide an option to add a confirmation dialog.
